dockerspawner/dockerspawner.py

```diff
@@ -442,6 +442,7 @@ def _default_name_template(self):
     )
 
     mounts = List(
+        Dict,
```

Comment:
should be an instance instead of a type:

```suggestion
        Dict(),
```